### PR TITLE
Added firebaseapp.com checks

### PR DESCRIPTION
**Pull request by six2dez**

### CODE REVIEW

**Comment by initstring** [comment]
Thank you for the PR!

I'm currently on holiday and trying not to spend too much time on the computer. We'll see how that goes. :)

It may take me up to a month until I can review this, I hope you don't mind.

Thanks!!!

**Comment by six2dez** [comment]
Np at all!!! :)

File: enum_tools/gcp_checks.py [hunk 1]
```diff
@@ -138,7 +139,55 @@ def check_fbrtdb(names, threads):
...
 
+    # Start a counter to report on elapsed time
+    start_time = utils.start_timer()
+
+    # Initialize the list of correctly formatted urls
+    candidates = []
+
+    # Take each mutated keyword craft a url with the correct format
+    for name in names:
+        # Firebase RTDB names cannot include a period. We'll exlcude
```
Comment (initstring) [inline enum_tools/gcp_checks.py:178]: Can you fix this comment to be about FB Apps. Also, can you confirm this is also the case for firebase apps?

**Comment by initstring** [comment]
@six2dez 

Besides my one small comment, this all looks good to me. Can you adjust the README to add Firebase Apps to the bulleted list of GCP resources in this PR?

Thanks!

**Comment by six2dez** [comment]
Done!

**Comment by initstring** [comment]
Thanks again for this contribution! Looks good to me, merging.

**Pull request merged**